tests/nlu/test_train.py

```diff
@@ -26,8 +26,11 @@ def get_test_params_for_component(component: Text) -> Dict[Text, Union[Text, int
     )
 
 
-def as_pipeline(*components):
-    return [{**{"name": c}, **get_test_params_for_component(c)} for c in components]
+def as_pipeline(*components) -> List[Dict[Text, Dict]]:
+    return [
+        {**{"name": c}, **get_test_params_for_component(c)} if isinstance(c, str) else c
```

Comment:
I think the first `**` is unneeded and `"name": c` can just be inlined.

Reply:
This change was generated by black but I agree that we don't need the first `**`.